CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add an additional boolean parameter to `XpringClient`'s constructor which allows toggling between protocol buffer implementations.
+- Switch the protocol buffer generator to use gRPC-Web instead of gRPC-Node when on browser (or when forcing HTTP) for cross environment compatability with web and node.
+- Add XMLHttpRequest to the global namespace for compatability with gRPC-Web when forcing HTTP on node.
```

Comment:
Would we ever want to force HTTP on node? If we're supporting `gRPC-Node` as well, do we really need the `XMLHttpRequest` global in Node-land?

Reply:
Well, the main reason I kept it was so I could use our test suite to test the web configuration also. Not sure how we would want to test this going forward without these two options.